### ISSUE TITLE
FP in generic.secrets.security.detected-artifactory-password

### ISSUE THREAD

**Issue by minusworld** [returntocorp/sgrep-rules#1178]
**Describe the bug**
From a user on Slack:
btw, the secrets regexes is firing a bunch of false positives for us but thankfully, I think, the false positives are a good example of why semgrep is better than Yelp's original detect secrets mechanism.
https://semgrep.dev/editor?registry=generic.secrets.security.detected-artifactory-password
fires on package-lock.json files like
```
      "integrity": "sha512-12345678APAjZas32d1f32a1sd2fasdfasdf32a1sd32f1a3s2d1f...=="
```
because of the AP..
One option is to exclude package-lock.json from all of the secrets regexes; another is to modify the rule to say pattern-not-in "sha\d+-(base64string)

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [x] P2: annoying but not blocking me
